frontends/web/.eslintrc.json

```diff
@@ -21,7 +31,21 @@
     "arrow-spacing": "error",
     "space-infix-ops": "error",
     "react/no-unused-prop-types": "error",
-    "react/jsx-equals-spacing": ["error", "never"]
+    "react/jsx-equals-spacing": ["error", "never"],
+    "react/react-in-jsx-scope": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "no-case-declarations": "off",
+    "react/no-children-prop": "off",
+    "prefer-const": "off",
+    "react/prop-types": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "no-extra-boolean-cast": "off",
+    "@typescript-eslint/ban-types": "off",
+    "no-undef": "off",
+    "no-empty": "off",
+    "react/display-name": "off",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "no-async-promise-executor": "off"
```

Comment:
Extra explanation: we had to add this in order to not change anything in our codebase (to make linter happy basically). Our previous eslint extension (that we removed) most likely had these already, so we didn't need to declare it here.